modules/qna/src/views/FormModal.jsx

```diff
@@ -128,7 +128,7 @@ export default class FormModal extends Component {
     } = this.props
 
     return this.props.bp.axios
-      .put(`/mod/qna/${this.props.id}`, this.state.item, {
+      .put(`/mod/qna/questions/${this.props.id}`, this.state.item, {
```

Comment:
Ah, it is an Id indeed (reply to my first comment)

Reply:
will rename for clarity